docker-compose.yaml

```diff
@@ -2,9 +2,16 @@ services:
   ffc-pay-statement-receiver:
     build:
       context: .
-      target: production
+      target: development
     image: ffc-pay-statement-receiver
     container_name: ffc-pay-statement-receiver
+    depends_on:
+      - ffc-pay-statement-receiver-redis
     environment:
       APPINSIGHTS_CLOUDROLE: ffc-pay-statement-receiver-local
       APPINSIGHTS_INSTRUMENTATIONKEY: ${APPINSIGHTS_INSTRUMENTATIONKEY}
+      REDIS_HOSTNAME: ffc-pay-statement-receiver-redis
+      REDIS_PASSWORD: redis
+
+  ffc-pay-statement-receiver-redis:
```

Comment:
As this is used in testing, could it be moved to the `override` file.  Also suggest binding it's port to localhost so Redis clients can be used to interact with data.

Also suggest binding a Docker Volume so cache can be persisted between starts.

Something like this:

```
    volumes:
      - redis_data:/data
    ports:
      - "6379:6379"
```

Though pick a non-clashing port with other payment services that use Redis, which I think is only Request Editor.